integration_tests/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Will this ever be published? If not, consider adding `"private": true`.

Reply:
Added in next commit along with eslint